src/auth/device-auth-flow.ts

```diff
@@ -207,6 +207,32 @@ export async function refreshAccessToken(selectedScopes?: string[]): Promise<str
   }
 }
 
+export async function revokeRefreshToken() {
+  try {
+    log('Attempting to revoke refresh token');
+
+    const refreshToken = await keychain.getRefreshToken();
+    if (!refreshToken) {
+      log('No refresh token found `in keychain');
+      return;
+    }
+    const config = getConfig();
+    const response = await fetch(`https://${config.tenant}/oauth/revoke`, {
```

Comment:
Doesn't seem like the function doesn't checks if the revocation was successful. It should probably at least check the HTTP status code.

Reply:
I'm not against it, but I didn't think it'd bring any value. 
Either the token expired, is invalid or succeeds.  Regardless of what happens, there's no following action here.